### PR TITLE
Add ssh private key type and add type to static store

### PR DESCRIPTION
**Pull request by jefferai**

### CODE REVIEW

File: internal/credential/static/ssh_private_key_credential.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+	"github.com/hashicorp/boundary/internal/oplog"
+	wrapping "github.com/hashicorp/go-kms-wrapping/v2"
+	"github.com/hashicorp/go-kms-wrapping/v2/extras/structwrapping"
+	"golang.org/x/crypto/ssh"
+	"google.golang.org/protobuf/proto"
+)
+
+var _ credential.Static = (*SshPrivateKeyCredential)(nil)
+
+// A SshPrivateKeyCredential contains the credential with a username and password.
```
Comment (louisruch) [inline internal/credential/static/ssh_private_key_credential.go:19]:
```suggestion
// A SshPrivateKeyCredential contains the credential with a username and private key.
```

File: internal/credential/static/ssh_private_key_credential.go [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+// It is owned by a credential store.
+type SshPrivateKeyCredential struct {
+	*store.SshPrivateKeyCredential
+	tableName string `gorm:"-"`
+}
+
+// NewSshPrivateKeyCredential creates a new in memory static Credential containing a
+// username and password that is assigned to storeId. Name and description are the only
```
Comment (louisruch) [inline internal/credential/static/ssh_private_key_credential.go:27]:
```suggestion
// username and private key that is assigned to storeId. Name and description are the only
```

**Review by louisruch — APPROVED** [review]
Couple of minor comment updates but otherwise LGTM!

**Comment by louisruch** [comment]
Just needs a changelog, but we can do that before release as well

**Comment by jefferai** [comment]
Thanks!

**Pull request merged**